docs/build/avalanchego-apis/contract-chain-c-chain-api.md

```diff
@@ -131,8 +131,7 @@ eth_baseFee() -> {}
 curl -X POST --data '{
     "jsonrpc":"2.0",
     "id"     :1,
-    "method" :"eth_baseFee",
-    "params" :{}
+    "method" :"eth_baseFee"
```

Comment:
It turns out that `"params" :[]` should be used for empty value instead of `"params":{}`. Can you please make this change in the doc?

Reply:
Sure. Done, and successfully tested.